packages/typegen/src/generate/query.ts

```diff
@@ -110,18 +110,28 @@ function generateForMeta (registry: Registry, meta: Metadata, dest: string, extr
       {
         file: 'rxjs',
         types: ['Observable']
+      },
+      {
+        file: '@polkadot/api/types',
+        types: ['ApiTypes']
       }
     ]);
     const interfaceStart = [
       "declare module '@polkadot/api/types/storage' {",
       indent(2)('export interface AugmentedQueries<ApiType> {\n')
     ].join('\n');
-    const interfaceEnd = `\n${indent(2)('}')}\n}`;
+    const interfaceEnd = `\n${indent(2)('}')}\n\n`;
+    const queryableStorageInterface = indent(2)('export interface QueryableStorage<ApiType extends ApiTypes> extends AugmentedQueries<ApiType> {')
```

Comment:
Weird, not seeing this appear in the generated. So probably will be fixed on CI with a generate run. No harm.